build.sbt

```diff
@@ -28,6 +30,8 @@ lazy val commonSettings = Seq(
       "-unchecked"),
   libraryDependencies ++= commonDeps)
 
+publish / skip := true
+
 lazy val serializer = (projectMatrix in file("serializer"))
```

Comment:
While I remember... I'd suggest renaming folders and lazy val here and for `codecs` so that everything is consistent with project name (so folder `borer-akka-serializer` and lazy val `borerAkkaSerializer`)... can be done on a follow-up PR

Reply:
Yea, I was about to do a follow-up PR: #48

Reply:
:spurdo-thumbs-up: